### ISSUE TITLE
Unintentionally removes /me

### ISSUE THREAD

**Issue by gizmogoat** [lunakittyyy/CommandsEx#1]
Writing an issue so I can have my thoughts written down for self reference later.

There's three main ways we can go about fixing this:
1. Adding an optional colon parameter to the _send_message func, replacing manually setting the bool in the send flow
2. Resend a message and manually apply the changes colon does
3. Move the point of injection past the point the base game commands happen

Option 1 is probably most optimal if we wish to replace things within the context of the CommandsEx API. This provides minor additional functionality to developers too as a bonus.

Option 2 would work, but its janky and duplicates larger portions of code for no good reason

Option 3 would also function, but our commands can and will conflict with the base game commands - plus, the base game commands won't show in the help menu unless we hardcode additional strings to be displayed in that menu.

**Comment by appdoze** [comment]
func helpRecieve(text, args):
    Network._update_chat("Commands:\n")
    var children = cExNode.get_children()
    var helpOut = "[color=red]"
    for i in children:
        if i.name.begins_with("ignore_"): continue
        helpOut = helpOut + i.get("commandText") + " "
    helpOut = helpOut + "[/color]"  # Close the red color tag
    helpOut = helpOut + "[color]Normal text color restored.[/color]"  # Reset to normal color
    Network._update_chat(helpOut)

your help command keeps the entire chat red - so this might be a fix

**Comment by gizmogoat** [comment]
> func helpRecieve(text, args): Network._update_chat("Commands:\n") var children = cExNode.get_children() var helpOut = "[color=red]" for i in children: if i.name.begins_with("ignore_"): continue helpOut = helpOut + i.get("commandText") + " " helpOut = helpOut + "[/color]" # Close the red color tag helpOut = helpOut + "[color]Normal text color restored.[/color]" # Reset to normal color Network._update_chat(helpOut)
> 
> your help command keeps the entire chat red - so this might be a fix

Are you sure? 

https://github.com/lunakittyyy/CommandsEx/blob/c639f0a88f5d728e05f57a0b89f3590583de89d3/mods/CommandsEx/basecommands.gd#L22-L30

Here I'm pretty explicitly closing the color tag at line 29. Is the whole chat red for you?

**Comment by appdoze** [comment]
yes the whole chat is red after using the help command, i''ll test it again alr so i tested it again and the whole chat is not red it might of been a bug earlier idk

**Comment by qwertyyuiopasdfghjklzxcvbnmqq** [comment]
also could you update CommandsEx so i can change the prefix of the slash '/' to a different prefix? :)

**Comment by appdoze** [comment]
also could you update CommandsEx so i can change the prefix of the slash '/' to a different prefix? :)
